enterprise/views.py

```diff
@@ -133,11 +144,11 @@ class GrantDataSharingPermissions(View):
         "opportunities from the world's best universities."
     )
 
-    def get_default_context(self, enterprise_customer, platform_name):
+    def get_default_context(self, enterprise_customer, platform_name, request):
```

Comment:
You could get rid of `platform_name` as a parameter here and pull it from the global context.